### DISCUSSION TITLE
How do you get the video thumbnail?

### DISCUSSION THREAD

**Discussion by restorepro** [Tyrrrz/YoutubeExplode#585] [Q&A]
I recently updated youtubeExplode and didn't see any documentation on getting the video thumbnail.
This is how it was obtained prior to the update:

youtubeImg.Source = video.Thumbnails.StandardResUrl;

Thanks!

**Comment by Tyrrrz** [comment]
Hi. Thumbnails are now provided as a collection:

https://github.com/Tyrrrz/YoutubeExplode/blob/349ca992c3dca5812bf793ff1d2c719a953b3996/YoutubeExplode/Videos/Video.cs#L39

You can use LINQ to get a specific thumbnail, or an extension method `GetWithHighestResolution()` that returns the best thumbnail:

https://github.com/Tyrrrz/YoutubeExplode/blob/349ca992c3dca5812bf793ff1d2c719a953b3996/YoutubeExplode/Common/Thumbnail.cs#L60-L65

**Reply by restorepro** [reply]
Ok, thanks for the reply sir. I will try it out.
